src/textual/css/stylesheet.py

```diff
@@ -433,6 +451,40 @@ def apply(
                 for name, specificity_rules in rule_attributes.items()
             },
         )
+
+        # Set initial values
```

Comment:
What “initial” is this? English word or `initial` value?
Are we replacing special values `initial` with an actual CSS value?

Could we rephrase the comment as something like what you can see below?

```suggestion
        # Replace the values `initial` with the appropriate default
```